### DISCUSSION TITLE
First Run of a Scheduled DAG

### DISCUSSION THREAD

**Discussion by Shlomixg** [apache/airflow#41477] [Q&A]
Hi,

I am experiencing an issue with Apache Airflow related to DAG scheduling. Specifically, I have a DAG configured to run on a weekly schedule with catchup=False, but it starts running immediately upon creation, rather than waiting for the next scheduled interval.

Here are the details of my DAG configuration:

```python
start_date=datetime(2024, 01, 01), # (Set to the creation date of the system)
schedule_interval="{"schedule": "00 10 * * 6"}" # weekly on Saturday
catchup=False,
max_active_runs=1,
is_paused_upon_creation=False,
```
Despite setting catchup=False, the DAG is triggered for the current date and time as soon as it is created. This behavior is not aligned with my expectation to have the DAG start from the next scheduled interval (i.e., from August 17, 2024).

I have tried the following approaches to address the issue:

Ensured catchup=False is set.
Considered other methods such as manually pausing the DAG, but it is not feasible in my scenario.
Is there a way to ensure that the DAG does not run until the next scheduled interval, even if it's created before the first run date?
Since it's a system that created multiple times we can't set a future start date.

I would appreciate any guidance or suggestions on how to handle this situation.
Thanks for your help!

Shlomi

**Comment by ahmedelsideeg** [comment]
Hello @Shlomixg 

I think you need to add this environment variable  

AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION=True
